05_iigo/index.tex

```diff
@@ -467,16 +467,28 @@ \section{IIGO Session order}
 
 \section{Future Work}
 
-\begin{itemize}
-    \item \textbf{Diplomatic sanctions}: Although having the potential of being a good alternative for severer sanctions discussed in  Section~\ref{sec:sanctions}, diplomatic sanctions are \emph{not} implemented within the scope of the coursework. \\
+\textbf{Diplomatic sanctions.} Although having the potential of being a good alternative for severer sanctions discussed in  Section~\ref{sec:sanctions}, diplomatic sanctions are \emph{not} implemented within the scope of the coursework. \\
     Suggested diplomatic sanctions include:
         \begin{itemize}
             \item Revoking an island's eligibility to vote and to be elected for a position.
-            \item Revoking an island's eligibility to propose a rule/motion.
+            \item Revoking an island's eligibility to propose any rule or a specific rule.
         \end{itemize}
-    \item \textbf{Immutable rules}: Within the scope of the coursework, a subset of rules could have been categorised as immutable. This means that to change such immutable rules, the islands first need to vote to change their status to be \emph{mutable}, and consequently, hold another vote to change these mutable rules.
+\textbf{Immutable rules.} Within the scope of the coursework, a subset of rules could have been categorised as immutable. This means that to change such immutable rules, the islands first need to vote to change their status to be \emph{mutable}, and consequently, hold another vote to change these mutable rules. An example of such an immutable rule could have been "An island in the archipelago cannot hold multiple IIGO roles (i.e. the President, Speaker, and Judge) in the same turn if at least two other islands are still alive.". This immutable rule would allow islands to avoid the tyranny that might arise from the lack of separation of power in any turn.\\
     %\item \textbf{Adding rules to the proposal list: } 
-\end{itemize}
+\textbf{Scope of IIGO.} Within the scope of the coursework, additional rules could be defined to audit levels of interaction at the IITO, IIFO and foraging. \\
+    Suggested additional rules include:
+        \begin{itemize}
+            \item During an IITO session, each island is not \emph{permitted} to give gifts to a sanctioned island.
```

Comment:
Maybe move to sanction expansion?

Reply:
there is no sanction expansion

Reply:
I don't think we should create one

Reply:
again, if you wanna introduce more sections in the future work --
we can merge this PR --> AND THEN you can revise sections or smth...
I cannot possibly know how many sections you want for future work if you cannot tell me